ipa-core/src/ff/boolean.rs

```diff
@@ -30,12 +26,9 @@ impl Boolean {
     }
 }
 
-impl ExtendableField for Boolean {
-    type ExtendedField = Gf32Bit;
-
-    fn to_extended(&self) -> Self::ExtendedField {
-        Gf32Bit::try_from(self.as_u128()).unwrap()
-    }
+impl PrimeField for Boolean {
```

Comment:
I wonder if this could hurt us in the future - not in terms of correctness, but performance-wise. Doing inefficient multiplications or any other specialized protocol that assumes prime fields to be large. @andyleiserson?

Reply:
I think this only impacts the attribution logic which we anyway plan to depreciate. Everything else is using vectorization.

Reply:
Here is another option: https://github.com/andyleiserson/ipa/commit/0e7307ef8bdc6745c142c65b415313b10a317b82

It is a little weird to implement PrimeField here, since it was mostly used with larger fields in mind. On the other hand, I don't think there's any real impact. It's only used in old protocols that OPRF IPA isn't using. And it's a bit weird to implement ExtendableField too -- what is the context where we want to treat Boolean as a field embedding in Gf32Bit? But we can't implement neither, without more involved changes.